CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ any parts of the framework not mentioned in the documentation should generally b
 
 * Removed support for Django 1.11.
 * Removed support for Django 2.1.
+* Removed support for Django REST framework 3.10, 3.11
+
+### Added
+* Added support for Django REST framework 3.12
```

Comment:
in release 3.2.0 below please also note, that that will be the last release supporting DRF 3.10 and DRF 3.11